src/packer.pkr.hcl

```diff
@@ -142,13 +150,191 @@ build {
+  //   # Install networking packages
+  //   script           = "src/powershell/install-category.ps1"
+  //   environment_vars = ["CATEGORY=networking"]
+  // }
+
+  // provisioner "windows-restart" {
+  //   # Wait a maximum of 30 minutes for Windows to restart.
+  //   # The build will fail if the restart process takes longer than 30 minutes.
+  //   restart_timeout = "30m"
+  // }
+
+  // provisioner "powershell" {
+  //   # Install password packages
+  //   script           = "src/powershell/install-category.ps1"
+  //   environment_vars = ["CATEGORY=passwords"]
+  // }
+
+  // provisioner "windows-restart" {
+  //   # Wait a maximum of 30 minutes for Windows to restart.
+  //   # The build will fail if the restart process takes longer than 30 minutes.
+  //   restart_timeout = "30m"
+  // }
+
+  // provisioner "powershell" {
+  //   # Install reverse engineering packages
+  //   script           = "src/powershell/install-category.ps1"
+  //   environment_vars = ["CATEGORY=reverse-engineering"]
+  // }
+
+  // provisioner "windows-restart" {
+  //   # Wait a maximum of 30 minutes for Windows to restart.
+  //   # The build will fail if the restart process takes longer than 30 minutes.
+  //   restart_timeout = "30m"
+  // }
+
+  // provisioner "powershell" {
+  //   # Install utility packages
+  //   script           = "src/powershell/install-category.ps1"
+  //   environment_vars = ["CATEGORY=utilities"]
+  // }
+
+  // provisioner "windows-restart" {
+  //   # Wait a maximum of 30 minutes for Windows to restart.
+  //   # The build will fail if the restart process takes longer than 30 minutes.
+  //   restart_timeout = "30m"
+  // }
+
+  // provisioner "powershell" {
+  //   # Install vulnerability analysis packages
+  //   script           = "src/powershell/install-category.ps1"
+  //   environment_vars = ["CATEGORY=vulnerability-analysis"]
+  // }
+
+  // provisioner "windows-restart" {
+  //   # Wait a maximum of 30 minutes for Windows to restart.
+  //   # The build will fail if the restart process takes longer than 30 minutes.
+  //   restart_timeout = "30m"
+  // }
+
+  // provisioner "powershell" {
+  //   # Install web application packages
+  //   script           = "src/powershell/install-category.ps1"
+  //   environment_vars = ["CATEGORY=web-applications"]
+  // }
+
+  // provisioner "windows-restart" {
+  //   # Wait a maximum of 30 minutes for Windows to restart.
+  //   # The build will fail if the restart process takes longer than 30 minutes.
+  //   restart_timeout = "30m"
+  // }
+
+  // provisioner "powershell" {
+  //   # Install wordlist packages
+  //   script           = "src/powershell/install-category.ps1"
+  //   environment_vars = ["CATEGORY=wordlists"]
+  // }
+
+  // provisioner "windows-restart" {
+  //   # Wait a maximum of 30 minutes for Windows to restart.
+  //   # The build will fail if the restart process takes longer than 30 minutes.
+  //   restart_timeout = "30m"
+  // }
```

Comment:
Is all this code that is commented out necessary?  We usually delete such code, since it can be revived from source control if necessary.

Reply:
Hi @jsf9k, for context, I'm currently working through an issue where we run out of disk space to install all the software in these commented out provisioners. I commented them out so we don't run out of disk space and I'm hopefully able to have a successful build so I can continue debugging. These will be uncommented one by one once we figure out a fix.

Reply:
@jsf9k this should be resolved as I uncommented all those llines. Can I mark this as resolved? thanks